Javascript/script.js

```diff
@@ -0,0 +1,138 @@
+var firstnum=0,secondnum=0,operator,result=0;
```

Comment:
1. The variable name use in camelCase like firstNum, secondNum.
2. Always assigned some value to a variable eg. operator you have not assigned anything